source/docs/software/kinematics-and-odometry/differential-drive-odometry.rst

```diff
@@ -20,55 +23,60 @@ The ``DifferentialDriveOdometry`` class requires one mandatory argument and one
       // our starting pose is 5 meters along the long end of the field and in the
       // center of the field along the short end, facing forward.
       DifferentialDriveOdometry m_odometry = new DifferentialDriveOdometry(
-        getGyroHeading(), new Pose2d(5.0, 13.5, new Rotation2d()));
+        m_gyro.getRotation2d(),
+        m_leftEncoder.getDistance(), m_rightEncoder.getDistance(),
+        new Pose2d(5.0, 13.5, new Rotation2d()));
 
    .. code-tab:: c++
 
       // Creating my odometry object. Here,
       // our starting pose is 5 meters along the long end of the field and in the
       // center of the field along the short end, facing forward.
-      frc::DifferentialDriveOdometry m_odometry{GetGyroHeading(),
+      frc::DifferentialDriveOdometry m_odometry{
+        m_gyro.GetRotation2d(),
+        units::meter_t{m_leftEncoder.GetDistance()},
+        units::meter_t{m_rightEncoder.GetDistance()},
         frc::Pose2d{5_m, 13.5_m, 0_rad}};
 
 
 Updating the Robot Pose
 -----------------------
 The ``update`` method can be used to update the robot's position on the field. This method must be called periodically, preferably in the ``periodic()`` method of a :ref:`Subsystem <docs/software/commandbased/subsystems:Subsystems>`. The ``update`` method returns the new updated pose of the robot. This method takes in the gyro angle of the robot, along with the left encoder distance and right encoder distance.
 
-.. note:: The encoder distances in Java must be in meters. In C++, the units library can be used to represent the distance using any linear distance unit. If the robot is moving forward in a straight line, **both** distances (left and right) must be positive.
+.. note:: If the robot is moving forward in a straight line, **both** distances (left and right) must be increasing.
```

Comment:
This seems more confusing. Increasing in the positive or negative direction? Does it not matter?

Reply:
I'd changed it because "both distances must be positive" doesn't make a whole lot of sense. Let me think if there's a better way to phrase it

Reply:
Updated version:

```**both** distances (left and right) must be increasing positively -- the rate of change must be positive.```

How does that sound? Not sure if I like it.